CMSgov/design-system
# [WNMGDS-500] Add cmsds.config.js

### Summary
- Many of our script options do not change, i.e. `sourceDir`, `typescript`, `githubUrl`. 
- In order to simplify script usage and our CLI logic, we create a new `cmsds.config.js` file which contains these configurations.

### How to test
- Run CMSDS scripts on the core, changing the different options in `cmsds.config.js` and ensuring they are working
- Test with child DS as well. 
[ds-site-package.zip](https://github.com/CMSgov/design-system/files/4899672/ds-site-package.zip)

Follow similar steps as https://github.com/CMSgov/design-system/pull/731 to ensure a clean build in the child ds
